lib/livebook/config.ex

```diff
@@ -425,6 +425,14 @@ defmodule Livebook.Config do
   """
   def app_version(), do: @app_version
 
+  @doc """
+  This is the URL of the application repo that will be checked to see whether
+  some new version is available. Note: Currently this must be a github repo.
+  """
+  def app_repo_url() do
+    Application.get_env(:livebook, :app_repo_url, "https://api.github.com/repos/livebook-dev/livebook/releases/latest")
+  end
```

Comment:
I would expect `repo_url` to point to the repo main page. I would make this more specific:

```suggestion
  @doc """
  Returns the GitHub org/repo used where releases are created.
  """
  def github_release_repo() do
    Application.fetch_env!(:livebook, :github_release_repo)
  end
```

And we can set the default in `config/config.exs`:

```elixir
config :livebook,
  ...,
  github_release_repo: "livebook-dev/livebook"
```

Reply:
The change sounds reasonable. It also avoids the need for the `Note:` in the `@doc`. Thanks for the suggestion

Reply:
Changes done